src/components/Motion/hooks/useIsomorphicLayoutEffect.ts

```diff
@@ -0,0 +1,7 @@
+import { useEffect, useLayoutEffect } from 'react';
+import { canUseDom } from '../../../shared/utilities';
+
+// It's safe to use `useLayoutEffect` but the warning is annoying
```

Comment:
Removed

Reply:
FYI - the warnings appear in the prompt when running tests

    ```
 Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://reactjs.org/link/uselayouteffect-ssr for common fixes.
          at Tooltip (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Tooltip/Tooltip.tsx:22:5)
          at th
          at Cell (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/Cell/index.tsx:63:9)
          at Cell (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/Cell/index.tsx:63:9)
          at /Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/Cell/index.tsx:292:43
          at tr
          at HeaderRow (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/Header/HeaderRow.tsx:10:5)
          at thead
          at Header (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/Header/Header.tsx:80:5)
          at table
          at div
          at div
          at /Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/OcTable.tsx:74:8
          at /Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/OcTable.tsx:74:8
          at div
          at OcTable (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Internal/OcTable.tsx:93:9)
          at div
          at InternalTable (/Users/dylan/Desktop/source/dylankilgore/octuplefork/src/components/Table/Table.tsx:62:9)

```